docs/howtoguides/enable_realtime_kernel.md

```diff
@@ -1,21 +1,20 @@
 # How to enable Real-Time Kernel
 
 ```{caution}
-Real-Time Kernel is currently in beta.
+Real-Time Kernel is currently in beta and is only supported on 22.04.
```

Comment:
Oops, this is no longer beta from our perspective. You can also remove all instances of "--beta" in this file.
```suggestion
Real-Time Kernel is only supported on 22.04.
```

Reply:
Good to know! I've removed them all now